examples/tutorial/04_Working_with_Tabular_Data.ipynb

```diff
@@ -111,7 +111,7 @@
     "\n",
     "A HoloViews `Dataset` is similar to a HoloViews Element, without having any specific metadata that lets it visualize itself. A Dataset is useful for specifying a set of `Dimension`s that apply to the data, which will later be inherited by any visualizable elements that you create from the Dataset.\n",
```

Comment:
```suggestion
    "A HoloViews `Dataset` is similar to a HoloViews Element, but without having the full set of metadata lets it visualize itself. A Dataset is useful for declaring the set of `Dimension`s along which your data varies, which will later be inherited by any more specific visualizable elements that you create from the Dataset.\n",
```

Reply:
how about:
"but without having the full set of metadata that would let it visualize itself"

Reply:
Sure.

Reply:
don't know how to fix this from here. tried to change my local copy and push itbut running into git problems? Any advice on how to accomplish this change?

Reply:
Looks to me like it's already been done.

Reply:
is there anything left for me to do here?

Reply:
Looks done to me!